### PR TITLE
multiple pages : Add Korean Translation

### PR DESCRIPTION
**Pull request by JJByun**
chsh.md
cksum.md
clamscan.md
clang.md
clear.md
clementine.md
cloc.md
clockwork-cli.md

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by JJByun** [comment]
> I'll let someone who knows the language merge this.

@schneiderl Thanks for your review

File: pages.ko/common/chsh.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# chsh
+
+> 사용자 로그인 쉘을 변경합니다.
+
+- 쉘 변경하기:
+
+`chsh -s {{path/to/shell_binary}} {{사용자이름}}`
```
Comment (Proscream) [inline pages.ko/common/chsh.md]: `chsh -s {{경로/쉘_바이너리명}} {{사용자이름}}` 
would be more great

**Comment by Proscream** [comment]
@sbrl Thanks for mention. If he change the tokens, LGTM.

File: pages.ko/common/cksum.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# cksum
+
+> 파일의 바이트 개수나 CRC 무결성 검사를 계산합니다.
+> 알립니다, 오래된 UNIX 시스템은 CRC 검사가 다를 수 있습니다.
+
+- 바이트 단위의 사이즈와 파일이름의 32비트 무결성 검사를 보여줍니다:
+
+`cksum {{파일 이름}}`
```
Comment (Proscream) [inline pages.ko/common/cksum.md]: 파일 이름 보단 파일명 으로 쓰는게 더 통일성 있을 듯 :)

File: pages.ko/common/clear.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# clear
+
+> 터미널 화면을 지웁니다(clear).
+
+- 터미널 화면을 지웁니다 (bash 쉘에서 Control-L을 누르는 것과 같은 기능입니다):
+
+`clear`
+
```
Comment (Proscream) [inline pages.ko/common/clear.md]: 마지막줄 따로 출력 되는 거 보니 2줄 엔터한 듯?

File: pages.ko/common/clamscan.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`clamscan -r {{경로/디렉토리}}`
+
+- `stdin` 으로부터 데이터를 검사합니다:
+
+`{{명령어}} | clamscan -`
+
+- 바이러스 데이터베이스 파일 또는 파일 디렉토리 지정합니다:
+
+`clamscan --database {{경로/데이터베이스 파일 혹은 디렉토리}}`
```
Comment (Proscream) [inline pages.ko/common/clamscan.md]: `clamscan --database {{경로/데이터베이스_파일명_혹은_디렉토리명}}`

File: pages.ko/common/clamscan.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`clamscan --infected`
+
+- 검사한 리포트를 로그 파일로 내보냅니다:
+
+`clamscan --log {{경로/로그파일}}`
+
+- 특정 디렉토리로 감염된 파일을 보냅니다:
+
+`clamscan --move {{경로/감염된 디렉토리}}`
```
Comment (Proscream) [inline pages.ko/common/clamscan.md]: `clamscan --move {{경로/감염된_디렉토리}}`

File: pages.ko/common/clang.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# clang
+
+> C, C++ 그리고 Objective-C 소스 파일을 컴파일합니다.  GCC의 드롭인 대체로 사용할 수 있습니다. 
+> 더 많은 정보: <https://clang.llvm.org/docs/ClangCommandLineReference.html>.
+
+- 실행 가능한 바이너리 파일로 소스 코드를 컴파일합니다:
+
+`clang {{입력 소스.c}} -o {{출력 실행가능파일}}`
```
Comment (Proscream) [inline pages.ko/common/clang.md]: `clang {{입력_소스.c}} -o {{출력_실행가능파일}}`

File: pages.ko/common/clementine.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+> 현대적인 음악 플레이어이자 라이브러리 생성자.
+> 더 많은 정보: <https://www.clementine-player.org>.
+
+- Clementine 열기:
+
+`clementine`
+
+- 음악 파일을 실행합니다:
+
+`clementine {{url 혹은 경로/파일.ext}}`
```
Comment (Proscream) [inline pages.ko/common/clementine.md]: `clementine {{url_혹은_경로/파일.ext}}`

File: pages.ko/common/clang.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- 모든 에러와 경고 메시지를 출력하도록 활성화합니다:
+
+`clang {{입력 소스.c}} -Wall -o {{출력 실행가능파일}}`
```
Comment (Proscream) [inline pages.ko/common/clang.md]: `clang {{입력_소스.c}} -Wall -o {{출력_실행가능파일}}`

File: pages.ko/common/clang.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+- 소스 파일과 다른 경로에 있는 라이브러리를 포함합니다:
+
+`clang {{입력 소스.c}} -o {{출력 실행가능파일}} -I{{헤더 경로}} -L{{라이브러리 경로}} -l{{라이브러리명}}`
```
Comment (Proscream) [inline pages.ko/common/clang.md]: `clang {{입력_소스.c}} -o {{출력_실행가능파일}} -I{{헤더_경로}} -L{{라이브러리_경로}} -l{{라이브러리명}}`

File: pages.ko/common/clockwork-cli.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+`clockwork-cli`
+
+- 특정 프로젝트의 Clockwork 로그들을 모니터링 합니다:
+
+`clockwork-cli {{경로/디렉토리}}`
+
+- 여러개의 프로젝트의 Clockwork 로그들을 모니터링 합니다:
+
+`clockwork-cli {{경로/디렉토리1_경로/디렉토리2 …}}`
```
Comment (Proscream) [inline pages.ko/common/clockwork-cli.md]: `clockwork-cli {{경로/디렉토리1 경로/디렉토리2 …}}`
여기는 원문이 띄어쓰는거라서 띄어쓰는게 나아보임.

**Comment by sbrl** [comment]
Merged, since @Proscream has approved.

**Pull request merged**